src/main/java/jpass/ui/JPassFrame.java

```diff
@@ -267,25 +276,30 @@ public void refreshFrameTitle() {
      * @param selectTitle title to select, or {@code null} if nothing to select
      */
     public void refreshEntryTitleList(String selectTitle) {
-        this.entryTitleListModel.clear();
+        this.entryTitleTableModel.setRowCount(0);
         List<String> titles = this.model.getTitles();
+        List<String> creation_dates = this.model.getCreationDates();
+        List<String> modified_dates = this.model.getModifiedDates();
         Collections.sort(titles, String.CASE_INSENSITIVE_ORDER);
 
         String searchCriteria = this.searchPanel.getSearchCriteria();
+        int i = 0;
         for (String title : titles) {
             if (searchCriteria.isEmpty() || title.toLowerCase().contains(searchCriteria.toLowerCase())) {
-                this.entryTitleListModel.addElement(title);
+                this.entryTitleTableModel.addRow(new Object[] {title,creation_dates.get(i),modified_dates.get(i)});
```

Comment:
It would be nice if the date columns will have dynamic column width. Currently, all the three columns have equal size, regardless of their content.

E.g.:
![image](https://user-images.githubusercontent.com/8159164/88818088-9579cb80-d1be-11ea-8a43-db128db2a421.png)

Reply:
Is [this](https://github.com/gaborbata/jpass/pull/17/commits/29b4529aeb8180cb243f1de551cb94bc83bca15d) what you meant @gaborbata ? Now all three columns have dynamic width and in case, you can also adjust the width of a column manually with the mouse.

Reply:
@AndreaRe9 thanks, it is exactly what I suggested.